htdocs/societe/class/societe.class.php

```diff
@@ -1924,6 +1929,12 @@ function add_commercial(User $user, $commid)
 			{
 				dol_syslog(get_class($this)."::add_commercial Erreur");
 			}
+			else {
+				$this->commercial_modified = $commid;
```

Comment:
Parameters to a triggers must be passed with
$this->context=array('commercial_modified'=>$commid);